pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -32,26 +32,20 @@ makeWrapper() {
     for ((n = 2; n < ${#params[*]}; n += 1)); do
         p="${params[$n]}"
 
-        if test "$p" = "--set"; then
+        if [[ "$p" == "--set" ]]; then
```

Comment:
fair, the fallthrough is only useful for a single pair of arguments which could be worked around: https://github.com/NixOS/nixpkgs/pull/19061#discussion_r113685418